README.md

```diff
@@ -1,2 +1,105 @@
-# Budget-app
-A rails mobile web application where you can manage your budget. There is  a list of transactions associated with a category, so that you can see how much money you spent and on what.
+![](https://img.shields.io/badge/Favour-Ezeugwa-black?labelColor=green)
+
+# Budget-App
+
+> This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
+
+## Built With
+
+- Ruby on Rails [click here >>](https://guides.rubyonrails.org/)
+- Rubocop: [Rubocop](https://rubocop.org/)
+- UML schema diagram
+- Bootstrap
+- Ruby
+
+### Live demo
+
+[Live Demo Link](https://budget-app-fave.herokuapp.com/)
+
+[Loom video](https://www.loom.com/share/6121f54dee074f73a146067a8b00a6bc)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+```
+  Ruby
+  Ruby on Rails
+  PostgreSQL
+```
+
+### Setup
+
+Clone the repository and change the directory to blog
+
+```
+  $ git clone git@github.com:Favourezeugwa/Budget-app.git
+
+  $ cd Budget-app
+```
+
+### Install
+
+Install all project dependencies by running the command below
+
+```
+  $ bundle install
+```
+
+### Run the application
+
+```
+  Linux
+  $ rails server
+```
+
+```
+  Windows
+  $ rails server
+```
+
+### Visit the homepage of application
+
+```
+  $ http://localhost:3000/
+```
+
+### Check linter errors
+
+```
+  $ rubocop .
+```
+
+### Run tests
+
+```
+  $ Rspec spec
+```
+
+### Author
+
+👤 **Favour Amarachi Ezeugwa**
+
+- GitHub: [@Favourezeugwa](https://github.com/Favourezeugwa)
+- LinkedIn: [Favour Amarachi Ezeugwa](https://www.linkedin.com/in/favour-amarachi-ezeugwa-a5bb31149/)
+- Twitter:[@Favour_ezeugwa](https://twitter.com/Favour_ezeugwa)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Favourezeugwa/Budget-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to [Microverse](https://bit.ly/MicroverseTN) for giving me this project to work on
+
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
@Deepakdanger I will fix this